src/main/java/com/synopsys/integration/jenkins/detect/service/DetectArgumentService.java

```diff
@@ -103,7 +103,9 @@ private String escapeArgument(String argument, Function<String, String> escaper)
         String cleanedArg = argument;
         if (argument.startsWith("--") && argument.contains("=")) {
             String[] splitArgument = argument.split("=", 2);
-            cleanedArg = splitArgument[0] + "=" + escaper.apply(splitArgument[1]);
+            //The api token should not be escaped... if it contains "=" or "==" padding, it would cause probs.
+            String endArg = splitArgument[0].contains("blackduck.api.token") ? splitArgument[1] : escaper.apply(splitArgument[1]);
+            cleanedArg = splitArgument[0] + "=" + endArg;
```

Comment:
Wouldn't other property values (like names of things) possibly contain '=' and possibly require escaping ('cause they also contain spaces, say)? Would it be safer to trigger on the property name, rather than an '=' in its value?